backend/app/views/spree/admin/store_credits/_update.js.erb

```diff
@@ -0,0 +1,5 @@
+<% if success = flash.discard(:success) %>
```

Comment:
Is it possible to avoid driving the frontend display via evaluated js from the server? Maybe this could return json that was displayed via JS.